packages/coolprop/meta.yaml

```diff
@@ -1,25 +1,20 @@
 package:
   name: coolprop
-  version: 6.6.0
-  # Problem with our added pyproject.toml/setup.py
+  version: 7.2.0
+  # Now uses upstream scikit-build-core-based build; local pyproject.toml/setup.py no longer needed
   pinned: true
   top-level:
     - CoolProp
 
 source:
-  url: https://sourceforge.net/projects/coolprop/files/CoolProp/6.6.0/source/CoolProp_sources.zip
-  sha256: ba3077ad24b36617fd7ab24310ce646a65bcbc8fde47f8de128cde2c72124b84
+  url: https://sourceforge.net/projects/coolprop/files/CoolProp/7.2.0/source/CoolProp_sources.zip
+  sha256: 2bd601b5e06b8765ab77fa121e2b0f4087b249c721ddb8745fd3e65952329689
   extract_dir: source
 
-  extras:
-    - - extras/setup.py
-      - ./setup.py
-    - - extras/pyproject.toml
-      - ./pyproject.toml
```

Comment:
As these are not needed anymore, could you please remove them from the repository as well?